testConnector/README.md

```diff
@@ -0,0 +1,203 @@
+# Overview
+
+The following documentation outlines how to incorporate a Test Connector as part of your project. This connector is used
+for testing the Connector Deployment tools in the Vantiq server, but is also a good example to follow when creating new 
+connectors. It offers the ability to retrieve data from files or environment variables, which can be done either via 
+publish/select requests, or by setting source configurations to poll for data from files.
+
+In order to incorporate this Enterprise Connector (also known as an Extension Source), you will need to create the 
+Source in the Vantiq IDE. The documentation has been split into two parts, [Setting Up Your Machine](#machine) and 
+[Setting Up Your Vantiq IDE](#vantiq).
+
+# Prerequisites <a name="pre" id="pre"></a>
+
+An understanding of the VANTIQ Extension Source SDK is assumed. Please read the [Extension Source README.md](../README.md) 
+for more information.
+
+The user must [define the TestConnector Source implementation](../README.md#-defining-a-typeimplementation) in the 
+Vantiq IDE. For an example of the definition, please see the 
+[*testConnectorImpl.json*](src/test/resources/testConnectorImpl.json) file located in the *src/test/resources* directory.
+
+Additionally, an example project named *testConnectorExample.zip* can be found in the *src/test/resources* directory.
+
+*   In order to activate the polling feature, simply remove the source configuration comments.
```

Comment:
What are comments in JSON? JSON doesn't have comments

And I don't see any things to remove in your example def'n.